TestAssets/TestProjects/SimpleCache/SimpleCache.csproj

```diff
@@ -0,0 +1,7 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+    <ItemGroup>
+        <PackageReference Include="System.Private.Uri" Version="4.4.0-beta-24821-02" />
+        <PackageReference Include="Microsoft.NETCore.CoreDisTools" Version="1.0.1-prerelease-00001" />
```

Comment:
The cache step is used to prime the assemblies. the run step will only occur after publish with filter profile. The CoreDistTools are unmanaged native component. I do not see the need for a SimpleManaged test. Will add one when i do publish PR